src/plugins/console/public/lib/mappings/legacy_template.ts

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import type { IndicesGetTemplateResponse } from '@elastic/elasticsearch/lib/api/types';
+
+export class LegacyTemplate {
+  constructor(private data: string[] = []) {}
```

Comment:
nit: I see you set `data` through the `load` method. Why you define that property it in constructor? Valid for other "loaders"